### PR TITLE
Add copy/paste buttons on import/export modal

### PR DESCRIPTION
**Pull request by TreTuna**
This adds the copy to clipboard button and functionality.

Signed-off-by: Tré Ammatuna <tre@tretuna.com>

### CODE REVIEW

File: src/renderer/i18n/hu.js [hunk 1]
```diff
@@ -101,7 +101,11 @@ const Hungarian = {
     layoutMode: "Billentyűzetkiosztás szerkesztése",
     colormapMode: "Színtérkép szerkesztése",
     importExport: "Aktuális réteg importálása/exportálása",
-    importExportDescription: `Az alábbi adatok szabadon szerkeszthetőek, vagy máshova másolhatóak, hogy később vissza lehessen illeszteni. Ezek az adatok a Chrysalis belső adatszerkezetét tükrözik, óvatosan szerkessze.`
+    importExportDescription: `Az alábbi adatok szabadon szerkeszthetőek, vagy máshova másolhatóak, hogy később vissza lehessen illeszteni. Ezek az adatok a Chrysalis belső adatszerkezetét tükrözik, óvatosan szerkessze.`,
+    copyToClipboard: `Másolja a vágólapra`,
```
Comment (algernon) [inline src/renderer/i18n/hu.js]:
```suggestion
    copyToClipboard: `Másolás a vágólapra`,
```

File: src/renderer/i18n/hu.js [hunk 2]
```diff
@@ -101,7 +101,11 @@ const Hungarian = {
...
+    copySuccess: `Másolt!`,
```
Comment (algernon) [inline src/renderer/i18n/hu.js]:
```suggestion
    copySuccess: `Másolva!`,
```

File: src/renderer/i18n/hu.js [hunk 3]
```diff
@@ -101,7 +101,11 @@ const Hungarian = {
...
+    pasteFromClipboard: `Illessze be a vágólapból`,
```
Comment (algernon) [inline src/renderer/i18n/hu.js]:
```suggestion
    pasteFromClipboard: `Beillesztés a vágólapról`,
```

File: src/renderer/screens/Editor/ImportExportDialog.js [hunk 4]
```diff
@@ -26,18 +28,37 @@ export const ImportExportDialog = withSnackbar(props => {
           null,
           2
         );
-  const onConfirm = () => {
+
+  function onConfirm() {
```
Comment (algernon) [inline src/renderer/screens/Editor/ImportExportDialog.js]: Any reason for going from `const onConfirm = () => { ` to `function onConfirm() {`? (Apart from the latter being shorter)

Comment (TreTuna) [reply src/renderer/screens/Editor/ImportExportDialog.js]: In this particular circumstance, it doesn't really matter. I have always tended to lean towards the classic function declaration unless dealing with issues of `this` due to function hoisting, no arguments array in arrow functions, and in some browsers (which shouldn't be an issue in this case with Electron) arrow functions don't actually get their names in the stack trace.

Happy to go either way here though.

Comment (algernon) [reply src/renderer/screens/Editor/ImportExportDialog.js]: Lets leave them as functions then.

**Pull request merged**